python3/utils.py

```diff
@@ -575,11 +573,16 @@ def get_result(accession):
     else:  # is_analysis(accession)
         return ANALYSIS_RESULT
 
+def get_result_accession(accession):
+    if is_run(accession) or is_experiment(accession) or is_sample(accession):
+        return 'run_accession'
+    else:  # is_analysis(accession)
```

Comment:
i think best to check for is_analysis() too

Reply:
I added extra checks in these functions, but there are already checks in place at the start of the script so it shouldn't end up in any other branch.